test/unit/util/FetchUtil.test.ts

```diff
@@ -22,34 +61,26 @@ describe('FetchUtil', (): void => {
       }));
     }
 
-    it('errors if the status code is not 200.', async(): Promise<void> => {
-      mockFetch('Invalid URL!', 404);
-      await expect(fetchDataset(url, converter)).rejects.toThrow(`Unable to access data at ${url}`);
-      expect(fetchMock).toHaveBeenCalledWith(url);
-    });
-
-    it('errors if there is no content-type.', async(): Promise<void> => {
-      fetchMock.mockResolvedValueOnce({ url, text: (): any => '', status: 200, headers: { get: jest.fn() }});
-      await expect(fetchDataset(url, converter)).rejects.toThrow(`Unable to access data at ${url}`);
-      expect(fetchMock).toHaveBeenCalledWith(url);
-    });
-
-    it('returns a Representation with quads.', async(): Promise<void> => {
-      mockFetch('<http://test.com/s> <http://test.com/p> <http://test.com/o>.');
-      const representation = await fetchDataset(url, converter);
-      await expect(arrayifyStream(representation.data)).resolves.toEqual([
-        quad(namedNode('http://test.com/s'), namedNode('http://test.com/p'), namedNode('http://test.com/o')),
-      ]);
-    });
-
     it('accepts Response objects as input.', async(): Promise<void> => {
       mockFetch('<http://test.com/s> <http://test.com/p> <http://test.com/o>.');
       const response = await fetch(url);
       const body = await response.text();
-      const representation = await fetchDataset(response, converter, body);
+      const representation = await responseToDataset(response, converter, body);
       await expect(arrayifyStream(representation.data)).resolves.toEqual([
         quad(namedNode('http://test.com/s'), namedNode('http://test.com/p'), namedNode('http://test.com/o')),
       ]);
     });
+
+    it('errors if the status code is not 200.', async(): Promise<void> => {
+      mockFetch('Incorrect status!', 400);
+      await expect(responseToDataset(await fetch(url), converter)).rejects.toThrow(`Unable to access data at ${url}`);
```

Comment:
At this point it would be easier to just completely remove fetch and its mock from this test file and directly input the response object here that is generated in the above mock function.